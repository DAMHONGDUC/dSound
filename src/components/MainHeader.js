import { COLORS } from "constants/theme";
import { Image, View, Text, StyleSheet, TouchableOpacity } from "react-native";
import Feather from "react-native-vector-icons/Feather";

export default function MainHeader() {
  return (
    <View style={styles.container}>
      <Image
        style={{ width: 35, height: 35 }}
        source={require("assets/app_icon.png")}
      />
      <Text style={styles.text}>dSound</Text>
      <TouchableOpacity style={{ flex: 0.1 }}>
        <Feather name={"settings"} color={COLORS.black} size={22} solid />
      </TouchableOpacity>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    flexDirection: "row",
    justifyContent: "flex-start",
    alignItems: "center",
    padding: 10,
    backgroundColor: COLORS.white,
  },
  text: {
    flex: 0.9,
    marginLeft: 15,
    fontSize: 20,
    color: COLORS.black,
    fontWeight: "500",
  },
});
