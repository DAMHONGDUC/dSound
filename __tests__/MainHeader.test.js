import renderer from "react-test-renderer";
import MainHeader from "components/MainHeader";

it("BottomPlayer Snapshot ", () => {
  const tree = renderer.create(<MainHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
