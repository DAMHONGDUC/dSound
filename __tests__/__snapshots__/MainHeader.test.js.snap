// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomPlayer Snapshot  1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "flex-start",
      "paddingLeft": 15,
      "paddingRight": 5,
    }
  }
>
  <Image
    source={
      {
        "testUri": "../../../src/assets/app_icon.png",
      }
    }
    style={
      {
        "height": 35,
        "width": 35,
      }
    }
  />
  <Text
    style={
      {
        "color": "#000000",
        "flex": 0.9,
        "fontSize": 20,
        "fontWeight": "500",
        "marginLeft": 15,
      }
    }
  >
    dSound
  </Text>
  <View
    accessible={true}
    collapsable={false}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "flex": 0.1,
        "opacity": 1,
      }
    }
  >
    <Text
      allowFontScaling={false}
      selectable={false}
      solid={true}
      style={
        [
          {
            "color": "#000000",
            "fontSize": 22,
          },
          undefined,
          {
            "fontFamily": "Feather",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          {},
        ]
      }
    >
      ï‡“
    </Text>
  </View>
</View>
`;
